### ISSUE TITLE
node.startEditing() is missing

### ISSUE THREAD

**Issue by gswcm** [amsik/liquor-tree#48]
The [events](https://amsik.github.io/liquor-tree/#Events) demo references a feature of "editing a node" which internally calls method `startEditing()` off of `node` object that is passed to the event handler.

I neither can find such method in [Node API](https://amsik.github.io/liquor-tree/#Node-API) nor I see this feature functioning correctly. In fact, an error gets reported into console once "Edit" link is clicked.

**Comment by amsik** [comment]
Unfortunately API page is not perfect. But I have tried to show all possible opportunities in examples. I do not get any errors by clicking `Edit` in the events demo. Can you show me the stack trace and browser version. I will check.

Also explain please what does `feature functioning correctly` mean? How it should work in your understanding?

**Comment by gswcm** [comment]
I'm not sure what happened over night, but it works now as expected... and my expectation was to see some kind of "input field" where node `text` could be changed. Previously (just yesterday) clicking on "Edit" resulted to no response in UI but instead a reference error about non-existent method `startEditing()` was displayed in the console. 

I cannot re-create the scenario. Both times I used latest Opera browser. 

The issue can now be closed. I apologize for the confusion if it was caused by something on my end.
